src/Stratis.Bitcoin.Features.MemoryPool/MempoolValidator.cs

```diff
@@ -995,7 +995,7 @@ private bool IsCurrentForFeeEstimation()
         /// <param name="context">Current validation context.</param>
         private void CheckAllInputs(MempoolValidationContext context)
         {
-            ScriptVerify scriptVerifyFlags = ScriptVerify.Standard;
+            var scriptVerifyFlags = ScriptVerify.Standard;
```

Comment:
Please revert it back

Reply:
I am still busy merging. The merge did not give me the option to keep it so I need to do this in steps.

Reply:
@noescape00, why? This aligns with your commit 229aa5b0.

Reply:
Oh, yes. Forgot that it's enum